dom/src/main/scala/com/thoughtworks/binding/dom.scala

```diff
@@ -312,22 +312,27 @@ object dom {
           case tree@NodeBuffer(children) =>
             nodeSeq(children)
           case tree@Elem(UnprefixedName(label), attributes, _, children) =>
-            val idOption = attributes.collectFirst { case (UnprefixedName("id"), Text(id)) => id }
+            val idOption = findTextAttribute("local-id", attributes).orElse(findTextAttribute("id", attributes))
             val elementName = idOption match {
               case None => TermName(c.freshName("element"))
               case Some(id) => TermName(id).encodedName.toTermName
             }
 
             val attributeMountPoints = for {
-              (key, value) <- attributes
+              (key, value) <- attributes if {
+                key match {
+                  case UnprefixedName("local-id") => false
+                  case _ => true
+                }
+              }
             } yield {
               val attributeAccess = key match {
                 case UnprefixedName(localPart) =>
-                  val keyName = TermName(localPart)
+                  val keyName = TermName(localPart).encodedName.toTermName
```

Comment:
What is the purpose of `encodedName.toTermName`?

Reply:
Initial implementation (with `LocalIdDummyOps`) doesn't *compile* without it, because `local-id` property name contains dash (the problem is similar https://github.com/ThoughtWorksInc/Binding.scala/issues/21)
Final solution compiles without this, but I preserved this change as common support of dashed attributes.